protocols/netlink/netlink_writer.go

```diff
@@ -0,0 +1,227 @@
+package protocolnetlink
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/bio-routing/bio-rd/config"
+	bnet "github.com/bio-routing/bio-rd/net"
+	"github.com/bio-routing/bio-rd/route"
+	"github.com/bio-routing/bio-rd/routingtable"
+	"github.com/bio-routing/bio-rd/routingtable/filter"
+	log "github.com/sirupsen/logrus"
+	"github.com/vishvananda/netlink"
+)
+
+// NetlinkWriter is a locRIB subscriber which serializes routes from the locRIB to the Linux Kernel routing stack
+type NetlinkWriter struct {
+	options *config.Netlink
+	filter  *filter.Filter
+
+	// Routingtable for buffering, to ensure no double writes (a.k.a rtnetlink: file exists)
+	mu sync.RWMutex
+	pt map[bnet.Prefix][]*route.Path
```

Comment:
What does pt stand for?

Reply:
path table. I guess that i should rename this :smile: